### PR TITLE
CCP-1173 migrate shopgate project change header colors

### PR DESCRIPTION
**Pull request by dqdquint**
# Pull Request Template

## Description

Updates to work with new router. No longer need to use theme, and can instead update based current pathname.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] I updated the CHANGELOG.md

Must use new portal in AppBar component on branch PWA-1518-Add-AppBar-portals of the PWA.

### CODE REVIEW

**Comment by aaron-martin** [comment]
It appears that the following constants from frontend/constants.js are no longer used and should be removed.
HEADER_BACKGROUND_COLOR_DEFAULT
HEADER_COLOR_DEFAULT
SET_NAVIGATOR_BACKGROUND
SET_NAVIGATOR_COLOR

**Review by aaron-martin — CHANGES_REQUESTED** [review]
There are constants defined that are no longer used and should be removed.

**Review by aaron-martin — APPROVED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (Carsten89) [inline frontend/package.json:1]: we should add the correct PWA 6.1 peerDependencies here

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.2] - 2018-12-13
+* Updates for extension to work with new router. Removed usage of subscription stream and instead update based on current pathname.
```
Comment (Carsten89) [inline CHANGELOG.md]:
```suggestion
* Updates for extension to work with PWA 6.1. Removed usage of subscription stream and instead update based on current pathname.
```
externals don't know the term "new router"

File: extension-config.json [hunk 3]
```diff
@@ -1,17 +1,18 @@
 {
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (Carsten89) [inline extension-config.json]: for the other extension that we migrated already we did a major version increase. We should do here the same

File: frontend/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
   "name": "@shopgate-project/change-header-colors",
```
Comment (Carsten89) [inline frontend/package.json]:
```suggestion
  "name": "@shopgate/change-header-colors",
```

**Review by Carsten89 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (dqdquint) [reply CHANGELOG.md]: Changed wording

File: frontend/package.json (hunk 4 above)
Comment (dqdquint) [reply frontend/package.json]: Updated to 2.0.0

File: frontend/package.json (hunk 1 above)
Comment (dqdquint) [reply frontend/package.json:1]: Updated peerDependencies for pwa-common and pwa-core to 6.1.0

File: extension-config.json (hunk 3 above)
Comment (dqdquint) [reply extension-config.json]: Updated to version 2.0.0

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**